src/Validator/Id.php

```diff
@@ -56,9 +58,15 @@ public function isValid()
             }
         }
 
-        // Get the session id bits per character INI setting, using 5 if unavailable
-        $hashBitsPerChar = ini_get('session.sid_bits_per_character');
-        $hashBitsPerChar = is_numeric($hashBitsPerChar) ? (int) $hashBitsPerChar : 5;
+        if (PHP_VERSION_ID >= 80400) {
+            // PHP 8.4 deprecated session.sid_bits_per_character and set it hard to "4".
+            // Old (pre PHP 8.4) session IDs with a higher bitrate are still valid though.
+            $hashBitsPerChar = 6;
```

Comment:
@MatthiasKuehneEllerhold Is there any reason for not defaulting to 5 bits on PHP 8.4+ too ?

Reply:
Because the user may have set 6 in a previous PHP version and these old session ID would be invalid then.